### DISCUSSION TITLE
Prácticas

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#161] [Q&A]
Buenos días, ¿Creen que den un poco más de tempo para  las prácticas html?

**Comment by doguedogue** [comment]
Quizás si, aunque se van a acumular o desfasar junto con las de la primer semana, lo mejor es ir adelantando poco a poco.
Tienes dudas? vamos tu puedes!

**Comment by GloriaCoralCerecedo** [comment]
Es que aún sigo con las prácticas de front end me faltan los diseños, pero si intentare avanzar y no atrasarme

**Comment by DanyVeneno** [comment]
Hola Gloria por lo que veo la experiencia que hemos tenido desde el inicio en LaunchX estan dando tiempos extras y no perderse los lives por que ahi da muchas respuestas nos vemos en un rato. Buen dia

**Comment by JuanSalvador73** [comment]
Seguramente si darán mas tiempo para las prácticas, de hecho ya está pasando. Sin embargo, yo aconsejo no retrasarse mucho en ellas porque efectivamente como dicen arriba se irán acumulando y luego puede ser un problema. Yo apenas terminé la de la intro de FrontEnd este domingo, a las 3 de la mañana hora CDMX. 

Y eso para que no se me junte con la de los pasteles que ya la tenemos encima. Es sencilla. Además no puedo siempre estar en los "online", pero si veo las grabaciones después, generalmente el mismo día horas después.

Sique adelante. Saludos.

**Reply by MaribelMendezM** [reply]
cuando se entrega la practica de la página de los pasteles?

**Comment by GloriaCoralCerecedo** [comment]
Buenas noches, ya dijo el MC que la práctica de HTML se entrega el Viernes igual, y comprendo pero no sé si a algunos les parezca moverlo hasta el domingo o Lunes. Tampoco una semana porque es mucho. 

No es difícil pero para los que aún no terminan el front end, ya estoy por acabar estoy en el UI pero si si está un poco tedioso estoy buscando herramientas para facilitarlo en Figma.

**Comment by DanyVeneno** [comment]
Hola Gloria, si aparentemente parece sencillo pero en realidad implica inversión de tiempo y más cuando estamos aprendiendo a usar herramientas nuevas a mi tambien me cuesta figma  create una cuenta en Udemy y este curso esta gratuito en la plataforma https://www.udemy.com/course/uiux-design-figma/ Saludos buen dia

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias, igual si estoy avanzando aunque no creo que me quedo muy bien XD pero ay voy. Ya van día días que me desvelo jaja

**Comment by emonreal9604** [comment]
Probablemente si, pero igual el fin de semana puede ponerte al día, es entendible que en la semana por trabajo, estudio o ambas te puedas atrasar. Pero al menos esta practicas que están pidiendo no son tan tardadas.

**Comment by GloriaCoralCerecedo** [comment]
Apenas acabo de terminar el front me desvele 3 días, no será el mejor trabajo pero es que hice 4 diseños tanto en wirefrime como UI, dos de escritorio tanto para abogado y cliente y dos de móvil y uno tenía 5 frames y otro como 10, entonces por eso me tarde mucho y hasta ahora acabe. 

Espero que den hasta el domingo o el lunes pero trataré de terminarlo

**Reply by hackit92** [reply]
Disculpa que me meta, no es por desalentarte pero aunque es muy importante el wireframe y la UI/UX, me parece que para la práctica está muy sobrado lo que has desarrollado y me da la impresión de que aún no has comenzado con las prácticas de HTML/CSS... te recomendaría no exigirte demasiado en esa primera práctica porque podrías atrasarte en temas que serán cada vez más complejos a lo largo del módulo. Por otro lado, felicitaciones por tanto esfuerzo en tu práctica, seguramente debe lucir increíble =) Mucha suerte

**Reply by GloriaCoralCerecedo** [reply]
Si tienes razón eso me dije, suelo ser muy perfeccionista y aunque puede ser bueno igual el tiempo afecta, pero ya lo termine. Si creo que el diseño le falta pero hice mi esfuerzo muchas gracias de verdad

**Reply by MauricioGongoraLoria** [reply]
Es muy tentador tratar de abordar mucho en el primer caso, es mi primera vez haciendo ese tipo de ejercicios, pero la parte del buy persona, determinar el mercado, te puedes tardar semanas y es fundamental para que un proyecto funcione, (en eso si te puedo decir que tengo algo de experiencia en el desarrollo de proyectos de inversión, es lo que presentas al cliente), pero en este caso es muy subjetivo por que realmente no conoces a quien se lo vas a proponer; lo estas inventando, por lo mismo te da la flexibilidad y lo dijo el MC no hay respuesta correctas, considero que es son practicas para que conozcas las herramientas y tus limitaciones, mas que realmente presentar una trabajo de nivel profesional que en una semana o dos teniendo otras actividades es muy complicado y mientras te vas atrasando en las demás herramientas que el programa tiene para enseñarte, que también será muy difícil que en una semana las domines por completo.

**Reply by GloriaCoralCerecedo** [reply]
Sinceramente yo en mi carrera lleve parte de desarolla de software y sinceramente era mucho. Y claro lleva si procedimiento, documentación y análisis. 

Pero siempre intento esmerarme y me tardo. 

Pero gracias a esto aprendí varias herramientas como el balsamic nunca lo había utilizado el Figma masomenos si.
